README.md

```diff
@@ -11,12 +11,34 @@
 - CSS
 - Linters
 
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+Code editor
+
+### Setup
+
+git clone the project and open the index.html file!
 
 ## Authors
 
 👤 **Elisha Tetteyfio**
 
-- GitHub: [@githubhandle](https://github.com/Elisha-Tetteyfio)
+- GitHub: [@Elisha-Tetteyfio](https://github.com/Elisha-Tetteyfio)
+
```

Comment:
- [ OPTIONAL] Kindly, this is a group project. Let's consider including all the authors in the README file.